smolyay/test_function_class.py

```diff
@@ -0,0 +1,6478 @@
+import abc
```

Comment:
Please rename this file `benchmark.py` so that it can be imported like `smolyay.benchmark`.